### ISSUE TITLE
老师，如果在Win上配置了MMDET环境后，如何修改指令执行train.py呀？

### ISSUE THREAD

**Issue by BandyKenny** [ShiqiYu/libfacedetection.train#79]
CUDA_VISIBLE_DEVICES=0 bash tools/dist_train.sh ./configs/yunet_n.py 2 12345
这句指令在win上无法执行，查看bash文件不知道如何修改？
#!/usr/bin/env bash

CONFIG=$1
GPUS=$2
PORT=$3
NNODES=${NNODES:-1}
NODE_RANK=${NODE_RANK:-0}
PORT=${PORT:-29500}
MASTER_ADDR=${MASTER_ADDR:-"127.0.0.1"}

PYTHONPATH="$(dirname $0)/..":$PYTHONPATH \
python -m torch.distributed.launch \
    --nnodes=$NNODES \
    --node_rank=$NODE_RANK \
    --master_addr=$MASTER_ADDR \
    --nproc_per_node=$GPUS \
    --master_port=$PORT \
    $(dirname "$0")/train.py \
    $CONFIG \
    --seed 0 \
    --launcher pytorch${@:4}
    # --auto-resume \

**Comment by Wwupup** [comment]
> 

我没在win下测试过，你最好贴一下你的报错细节

**Comment by Wwupup** [comment]
@BandyKenny 你好，只有4GB显存的话，必须调小batch_size。`samper_per_gpu`是表示每张GPU上的batch_size
https://github.com/ShiqiYu/libfacedetection.train/blob/dce01651d44d2880bcbf4e296ad5ef383a5a611e/configs/yunet_n.py#L29-L32
![image](https://github.com/ShiqiYu/libfacedetection.train/assets/70632698/ef1ab7cd-2564-43dc-82ff-0c0d0978a4cb)
另外，出错时尽量别改成熟包的核心代码，考虑设置输入参数，否则容易越改越崩。

**Comment by BandyKenny** [comment]
> 

好的，就是说端口号这个和分布式计算相关，在单GPU计算时候就可以不关注这个量，随便赋值一个端口号即可，🙇‍

**Comment by BandyKenny** [comment]
> 但是这个“The system cannot find the path specified.”系统找不到指定路径
这个提示对于训练是否会有影响呀？
